### DISCUSSION TITLE
Github Octernship Program

### DISCUSSION THREAD

**Discussion by debanjo31** [community/community#49769] [GitHub Education] [answered]
I showed interest in one of the GitHub octernship projects, it directed me to a new repo with a readme that explains fully the task of what i am expected to do

Am I to fork this project to start working one?
or I am to create a new react app and sent it to the organization?

Thanks

**Comment by samuel-ma** [comment, marked as answer]
@debanjo31 
Yes, create a fork for the task and work on it in your local device
Once your done, create a pull request and wait for your review
Plz check the main repo, I gave u a response earlier [here](https://github.com/HousewareHQ/frontend-engineering-octernship/issues/4)
